documentation-site/components/version-selector.js

```diff
@@ -81,8 +81,11 @@ const VersionSelector = () => {
     <StatefulPopover
       placement={PopoverPlacement.bottomLeft}
       dismissOnClickOutside={false}
+      focusLock
+      // eslint-disable-next-line jsx-a11y/no-autofocus
+      autoFocus
       content={({close}) => (
-        <NestedMenus>
+        <NestedMenus onEsc={close}>
```

Comment:
I think `onEsc` on `NestedMenus` may not be required. I believe escape handling is not working here because of the changed focus element. You may want to reconfigure this to be a stateless popover if you need more control

Reply:
Ah, didn't notice that method- seems to do the trick. I'm wondering if its okay to have it be the default (esc resets parent menu). I pushed up changes to review.

Reply:
Yeah the new pop over created by maybe-child-menu doesn't get any `onEsc/Escape` handlers. But I think this works fine with reseting parent now. (Two escapes to close but makes sense).